packages/cli/src/commands/deploy/helper.ts

```diff
@@ -355,25 +357,60 @@ export const loadManifest = (): Manifest | undefined => {
   }
 }
 
-export const createFieldType: CreateFieldTypeFunc = async ({
-  name,
-  body,
-  client,
-  options,
-}) => {
-  const newFieldPlugin = await client.createFieldType({ name, body })
-
-  //since `options` and `publish` properties are only accepted during updates,
-  //we need to force an update call right after the creation.
-  //If no options is found, it's not going to be sent to the API since undefined
-  //properties are not encoded.
-  await client.updateFieldType({
-    id: newFieldPlugin.id,
-    publish: true,
-    field_type: {
-      options,
-    },
-  })
+export const createFieldPlugin = async (
+  client: StoryClientType,
```

Comment:
nit: I think having this as an object argument might be much better to prevent issues in the future. But it is just nitpicking on the highest level, so no need to adjust it :) 

https://matheus.ro/2018/01/29/clean-code-avoid-many-arguments-functions/